partner_second_lastname/models.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# © 2015 Grupo ESOC Ingeniería de Servicios, S.L.U.
+
+from openerp import api, fields, models
+from openerp.addons.partner_firstname import exceptions
+
+
+class ResPartner(models.Model):
+    """Adds a second last name."""
+
+    _inherit = "res.partner"
+
+    lastname2 = fields.Char("Second last name")
+
+    @api.one
+    @api.depends("firstname", "lastname", "lastname2")
+    def _compute_name(self):
```

Comment:
Actually I need to modify it because it depends in `lastname2` now. BTW, https://github.com/OCA/partner-contact/pull/159 complicated more than helped. I'll update and fix it here too ASAP.